lib/cc/services/github_pull_requests.rb

```diff
@@ -87,8 +86,7 @@ def presenter
   def update_status_error
     update_status(
       "error",
-      "Code Climate encountered an error while attempting to analyze this " +
-        "pull request."
+      message
```

Comment:
What do you think about `@payload["message"] || DEFAULT_ERROR_MESSAGE`?

I think generally, all the `update_status_x` methods should use `@payload["message"] || SOME_DEFAULT` and let the callers be in charge of it (not the presenter here), but that's a bigger change and we can punt on it for now.